c2corg_ui/templates/helpers/edit.html

```diff
@@ -50,9 +50,14 @@ <h3 class="heading show-phone" translate>summary</h3>
       </div>
       % endif
       ## QUALITY
-      <div class="full comment" ng-class="{'has-success': saveCtrl.quality}" ng-init="qualityTypes = ${quality_types}">
-        <label translate>document_quality</label>
-        <select ng-options="type for type in qualityTypes" ng-model="saveCtrl.quality" class="form-control"><option value=""></option></select>
+      <div class="full comment" ng-class="{'has-success': saveCtrl.quality}">
+        <label translate>quality</label>
+        <select ng-model="saveCtrl.quality" class="form-control">
+          <option value=""></option>
+          % for q in quality_types:
```

Comment:
> by changing the scope you mean create a scope.$new(true)

No clue. Perhaps I am wrong. But I have the feeling it should be possible to make the usual way work in your modal box.

Have you tried ``$scope.$parent`` or ``$rootScope`` or... ?
It seems that ``mainCtrl`` is not available in the scope provided to the template.

> Should I merge as is (because at least it works), awaiting a possible solution ?

There is no hurry.